### ISSUE TITLE
i64 is used for integers when i32 should be used

### ISSUE THREAD

**Issue by mathstuf** [graphql-rust/graphql-parser#53]
[This code](https://github.com/graphql-rust/graphql-parser/blob/8a759df14ff6a2d97b55bcbccad0a53ce0bee4a6/src/common.rs#L46) says that the spec doesn't have a size for numbers, but [it does](https://spec.graphql.org/draft/#sec-Int):

> The Int scalar type represents a signed 32-bit numeric non-fractional value. Response formats that support a 32-bit integer or a number type should use that type to represent this scalar.

See: graphql-rust/graphql-client#266

**Comment by tailhook** [comment]
This is true for inputs and outputs. But I'm still not sure it is true for literals.

**Comment by mitchrmiller** [comment]
I made a feature for this so it's not breaking and thus doesn't need to wait for a major version. Someone could even add deprecation notice or on move to v1.0.0 just switch it to default to i32 and still offer the i64 as a feature to make the transition easier for people. #65
